app/ante/codec.go

```diff
@@ -0,0 +1,16 @@
+package ante
+
+import (
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/types/tx"
+	ethermint "github.com/evmos/ethermint/types"
+)
+
+var Registry = codectypes.NewInterfaceRegistry()
+
+func init() {
+	Registry.RegisterImplementations(
```

Comment:
isn't it already registered here `ethermint.RegisterInterfaces(registry)`
https://github.com/dymensionxyz/dymension/pull/777/files#diff-e24003257de410df6ad072f598ff69c460940943f37abb272d8dae365e6036beL46